tests/ptf_runner.py

```diff
@@ -1,17 +1,22 @@
 import pipes
 import traceback
 import logging
+from datetime import datetime
 
 logger = logging.getLogger(__name__)
 
 
 def ptf_collect(host, log_file):
-    host.fetch(src=log_file, dest='./logs/ptf_collect/', flat=True, fail_on_missing=False)
     pos = log_file.rfind('.')
-    pcap_file = (log_file[0:pos] if pos > -1 else log_file) + '.pcap'
+    filename_prefix = log_file[0:pos] if pos > -1 else log_file
+    rename_prefix = filename_prefix.replace('/tmp/', '')
```

Comment:
Are all log files saved to /tmp/ folder？Is it better to split folder name and file name with "/"? Dest only cares file name, then we don't have to replace /tmp here.

Reply:
Yes, we don't have to replace /tmp/